README.md

```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
  - [彭于斌 (archibate)](https://github.com/archibate) (OpenGL Compute Shader backend)
  - [Mingkuan Xu (xumingkuan)](https://github.com/xumingkuan) (IR optimization & standardization)
 
-[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions. [[List of all contributors]](https://github.com/taichi-dev/taichi/graphs/contributors)
+[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions.
```

Comment:
Why not move KLozes and squarefk to the list above too?

Reply:
second this

Reply:
No feelings hurt here! You guys on the above lust have done ton more work than me!

Reply:
Thanks for the proposal and for understanding. I actually think, for now, the existing description achieves a good trade-off between *visibility* and *fairness*.

Of course, the list will be updated in the future whenever necessary.